packaging/conf/osinfo-defaults.properties

```diff
@@ -59,7 +59,9 @@ os.other.resources.minimum.numberOfCpus.value = 1
 os.other.cpu.hotplugSupport.value = true
 os.other.cpu.hotunplugSupport.value = false
 os.other.q35Support.value = true
-os.other.devices.display.protocols.value = spice/qxl,vnc/vga,vnc/qxl,vnc/bochs
+os.other.devices.display.protocols.value = vnc/vga,vnc/bochs,vnc/qxl,spice/qxl
```

Comment:
this change is not needed anymore, right?

Reply:
doesn't harm, is it? but yes, generally not needed.
do you wish to drop it?

Reply:
yes please, let's minimize the changes